src/DynamoCore/Graph/Connectors/ConnectorPinModel.cs

```diff
@@ -5,14 +5,41 @@
 
```

Comment:
@QilongTang are these the only two files changed that went missing from the previous PR?

Reply:
Yes, I checked and tested a bit, your other changes are there